builder/builder.go

```diff
@@ -24,12 +25,26 @@ type Builder struct {
 func New(ownAddr *felt.Felt, bc *blockchain.Blockchain, builderVM vm.VM, log utils.Logger) *Builder {
 	return &Builder{
 		ownAddress: *ownAddr,
-
-		bc: bc,
-		vm: builderVM,
+		bc:         bc,
+		vm:         builderVM,
 	}
 }
 
+func (b *Builder) Receipt(fee *felt.Felt, feeUnit core.FeeUnit, txHash *felt.Felt, reverted bool,
```

Comment:
It doesnt look like this needs to be a member function. Let's make it standalone?